src/main/scala/com/yotpo/metorikku/utils/HudiUtils.scala

```diff
@@ -0,0 +1,30 @@
+package com.yotpo.metorikku.utils
+
+import org.apache.hudi.avro.model.HoodieCompactionPlan
+import org.apache.hudi.common.table.{HoodieTableMetaClient, HoodieTimeline}
+import org.apache.hudi.common.table.timeline.HoodieInstant
+import org.apache.hudi.common.table.timeline.HoodieInstant.State
+import org.apache.hudi.common.util.CompactionUtils
+import org.apache.hudi.common.util.collection.ImmutablePair
+import org.apache.hudi.config.HoodieWriteConfig
+import org.apache.hudi.table.HoodieTable
+import org.apache.spark.SparkContext
+import org.apache.spark.api.java.JavaSparkContext
+
+object HudiUtils {
+  def deletePendingCompactions(sparkContext: SparkContext, basePath: String): Unit = {
+    val jsc = JavaSparkContext.fromSparkContext(sparkContext)
```

Comment:
You need to protect yourself on the first run as this will not work if the folders empty. In any case I think it’s good to wrap this in try catch